dev/com.ibm.ws.kernel.boot_test/test/com/ibm/ws/kernel/boot/BootstrapConfigTest.java

```diff
@@ -222,7 +222,7 @@ public void testConfigureBadLocation() throws IOException {
         file.deleteOnExit();
 
         TestBootstrapConfig bc = new TestBootstrapConfig();
-        bc.findLocations(testName.getMethodName(), file.getAbsolutePath(), null, null, null);
+        bc.findLocations(testName.getMethodName(), file.getAbsolutePath(), null, null, null, null);
```

Comment:
I'm unsure why these tests are all calling the extra param method to only pass in null when the original one does the same thing.

Reply:
Definitely overzealous.  I'll backout the unnecessary changes here.